### PR TITLE
Adding operations to Snowflake Makefile

### PR DESCRIPTION
**Pull request by tushartathgur**
- make fmt
- make tidy
- make golangci
- make golangci-fix
    
These operations help when dealing with new changes in Snowflake

### CODE REVIEW

File: snowflake/Makefile [hunk 1]
```diff
@@ -44,3 +49,28 @@ generate:
 .PHONY: clean
 clean:
 	rm -rf bin
```
Comment (yuntanghsu) [inline snowflake/Makefile:51]: remove-GOLANGCI_LINT_BINDIR as well?

**Review by yuntanghsu — COMMENTED** [review]
Are we able to add these changes into the workflows?

File: snowflake/Makefile (hunk 1 above)
Comment (tushartathgur) [reply snowflake/Makefile:51]: Thanks for pointing out!

**Comment by tushartathgur** [comment]
> Are we able to add these changes into the workflows?
we did not add these changes yet to workflow.

File: snowflake/Makefile [hunk 2]
```diff
@@ -43,4 +48,30 @@ generate:
...
+	@rm -rf $(GOLANGCI_LINT_BINDIR)
+
+.PHONY: tidy
+tidy:
+	@rm -f go.sum
+	@$(GO) mod tidy
+
+.PHONY: fmt
+fmt:
+	@echo
```
Comment (yuntanghsu) [inline snowflake/Makefile:61]: Extra echo?

File: .github/workflows/go.yml [hunk 3]
```diff
@@ -112,6 +112,25 @@ jobs:
...
+      uses: actions/checkout@v4
+    - name: Set up Go using version from go.mod
+      uses: actions/setup-go@v4
+      with:
+        go-version-file: 'snowflake/go.mod'
+        cache-dependency-path: '**/snowflake/go.sum'
+    - name: Run snowflake golangci-lint
+      run: cd snowflake ; make golangci
+
   tidy-manifest:
```
Comment (yuntanghsu) [inline .github/workflows/go.yml:134]: Maybe add `make test-tidy` for snowflake as well?

Comment (tushartathgur) [reply .github/workflows/go.yml:134]: Added, Thanks!

**Review by yuntanghsu — APPROVED** [review]
LGTM

**Pull request merged**